### ISSUE TITLE
Query annotations as in iNaturalist Search URL's

### ISSUE THREAD

**Issue by ZekeMarshall** [ropensci/rinat#53]
I would like to be able to use the rinat package to search for observations of a given species, which have the annotation value "Fruiting".

The iNaturalist export tool (https://forum.inaturalist.org/t/how-to-use-inaturalists-search-urls-wiki-part-2/18792) allows for you to pass the following additional terms into the query to achieve this: "&term_id=12", and "term_value_id=14", which represent the "Plant Phenology" and "Fruiting" terms respectively.

I was wondering whether this is something that can be achieved in the rinat package?

Thanks for all your work on the package, it's a great resource!

**Comment by stragu** [comment]
@ZekeMarshall : I have added the feature, available in [version 0.1.9](https://github.com/ropensci/rinat/releases/tag/v0.1.9).

The commit in question is 977a047.

You can now do:

``` r
get_inat_obs(query = "Lilly Pilly", annotation = c(12, 14))
```

Where 12 is the term id, and 14 is the value id.

It seems the old API version we are currently using only allows a single value id, unfortunately, but I'll remember to allow more than one value when we move to the more recent API.

Thanks for the idea!

**Comment by ZekeMarshall** [comment]
Hi @stragu , that's brilliant - thanks for your work in implementing this feature! This will be very useful!
